volumes/volume_state.go

```diff
@@ -15,6 +15,7 @@ import (
 )
 
 // Returns the current state and the configuration of a volume.
+// UUIDs can also be names.
 //
 // See: https://docs.kraft.cloud/006-rest-api-v1-volumes.html#state
 func (c *volumesClient) State(ctx context.Context, uuid string) (*Volume, error) {
```

Comment:
Rename `uuid string` to `uuidOrName string`

Reply:
not sure why it does not show as outdated, i removed the comment.

Volumes do not have the name field yet it seems. I will try adding it now.

Reply:
added now, will resolve